### ISSUE TITLE
feature: show open files in fuzzy finder

### ISSUE THREAD

**Issue by aleclarson** [atom/fuzzy-finder#264]
If any open files are not in the current project, they are not found by the fuzzy finder.

I would like to use the fuzzy-finder to switch to a file that is:
- open in the current Atom window
- but not inside the project root(s)

**Comment by 50Wliu** [comment]
cmd/ctrl-b will allow you to switch within open files.  Is that what you're asking for?

**Comment by aleclarson** [comment]
Yeah, thanks. Is there an option to have those results appear in the cmd-P prompt?

**Comment by 50Wliu** [comment]
Not at the moment, no.
